src/spatialdata/_io/io_points.py

```diff
@@ -29,8 +30,10 @@ def _read_points(
     assert isinstance(store, (str, Path))
     f = zarr.open(store, mode="r")
 
-    path = Path(f._store.path) / f.path / "points.parquet"
-    table = read_parquet(path)
+    path = os.path.join(f._store.path, f.path, "points.parquet")
+    # cache on remote file needed for parquet reader to work
+    # TODO: allow reading in the metadata without caching all the data
+    table = read_parquet("simplecache::" + path if "http" in path else path)
```

Comment:
googled this cause i didn't know what it should do and found this option https://github.com/catalyst-cooperative/pudl/issues/1496#issuecomment-1085033950 and https://github.com/intake/intake-parquet/issues/18#issuecomment-1092217888 , maybe better?

Reply:
I think we would need to benchmark that. I'd do that in a follow up PR. @berombau WDYT?